packages/vscode-extension/package.nls.json

```diff
@@ -229,6 +254,8 @@
     "teamstoolkit.handlers.defaultManifestTemplateNotExists": "Manifest template file not found in %s. Use CLI with your own template file.",
     "teamstoolkit.handlers.defaultAppPackageNotExists": "App package file not found in %s. Use CLI with your own app package file.",
     "teamstoolkit.localDebug.failedCheckers": "Unable to check: [%s].",
+    "teamstoolkit.handlers.askInstallOfficeAddinDependency": "Install dependencies for Office Add-in?",
+    "teamstoolkit.handlers.installOfficeAddinDependencyCancelled": "Installation of dependencies is cancelled. But you can install dependencies manually by clicking on the 'Development - Check and install dependencies' button on the left side.",
```

Comment:
Proposed: Dependency installation is canceled, but you can install dependencies manually by clicking the 'Development - Check and Install Dependencies' button on the left side.
Logic: Corrected punctuation and grammar errors.

Reply:
fixed.